Doc/whatsnew/3.13.rst

```diff
@@ -1052,6 +1058,30 @@ See :pep:`744` for more details.
 Tier 2 IR by Mark Shannon and Guido van Rossum.
 Tier 2 optimizer by Ken Jin.)
 
+.. _free-threaded-cpython:
+
+Free-threaded CPython
+=====================
+
+CPython will run with the :term:`global interpreter lock` disabled when
+configured using the ``--disable-gil`` option. This is an experimental feature
+and work is still ongoing: expect some bugs and a substantial single-threaded
+performance hit.
+
+* Use :func:`!sys._is_gil_enabled` to determine if the :term:`GIL` is enabled.
+
+* Use ``sysconfig.get_config_var("Py_GIL_DISABLED")`` to identify CPython
+  builds configured with ``--disable-gil``.
```

Comment:
It's not clear from this why we need both APIs. I'm guessing it's because even when built with `--disable-gil` the GIL may be dynamically re-enabled?